tools/api-markdown-documenter/CHANGELOG.md

```diff
@@ -32,7 +34,7 @@ const apiModel = await loadModel({
 
 const transformConfig = {
 	apiModel,
-	uriRoot: ".",
+	uriRoot: "",
```

Comment:
Should we remove it from the example?

Reply:
I went back and forth on this. Since this parameter was required _prior_ to the `0.18.0` changes, I figured it made sense to keep it in the "before" example. And since this particular example isn't about `uriRoot`, I figured it would be strange to remove it from the "after" example.

Reply:
But maybe I'll add a comment noting that the parameter is now optional...

Reply:
Updated!

Reply:
Didn't realize it's in the "before" example, I agree it still makes sense there.